src/components/common/modal/ModalWrapper.js

```diff
@@ -0,0 +1,186 @@
+      paddingRight: theme.modals.overlayHorizontalPadding,
+      paddingTop: theme.modals.overlayVerticalPadding,
+      width: '100vw',
+    },
+    modalCloseImageContainer: {
+      position: 'relative',
+      zIndex: mediumZIndex,
+    },
+    modalCloseImage: {
+      height: normalize(37),
+      marginBottom: -normalize(37 / 2),
+      marginLeft: 'auto',
+      marginRight: -normalize(37 / 2),
+      width: normalize(37),
+    },
+    modalContainer: {
+      display: 'flex',
+      flexDirection: 'row',
+      flexGrow: 1,
+      position: 'relative',
+      width: '100%',
+      zIndex: lowZIndex,
+    },
+    modalInnerContents: {
+      backgroundColor: theme.modals.backgroundColor,
+      borderBottomRightRadius: theme.modals.borderRadius,
+      borderTopRightRadius: theme.modals.borderRadius,
+      boxShadow: '0 19px 38px rgba(0, 0, 0, 0.5)',
+      flexGrow: 1,
+      padding: theme.modals.contentPadding,
+      position: 'relative',
+      zIndex: lowZIndex,
+    },
+    modalContainerStraightenBottomRightEdge: {
+      borderBottomRightRadius: '0',
+    },
+    modalLeftBorder: {
+      backgroundImage: `url(${wavePattern})`,
+      backgroundRepeat: 'repeat-Y',
+      borderBottomLeftRadius: theme.modals.borderRadius,
+      borderTopLeftRadius: theme.modals.borderRadius,
+      flexGrow: 1,
+      flexShrink: 0,
+      maxWidth: theme.modals.borderLeftWidth,
+      minWidth: theme.modals.borderLeftWidth,
+      position: 'relative',
+      zIndex: mediumZIndex,
+    },
+    modalLeftBorderAddMarginBottom: {
+      marginBottom: theme.modals.jaggedEdgeSize,
+    },
+    modalContents: {
+      flexGrow: 1,
+      flexShrink: 1,
+    },
+    jaggedEdge: {
+      backgroundImage: `linear-gradient(45deg, transparent 75%, ${
+        theme.modals.backgroundColor
+      } 76%), linear-gradient(-45deg, transparent 75%, ${theme.modals.backgroundColor} 76%)`,
+      backgroundPosition: '0 0',
+      backgroundRepeat: 'repeat-x',
+      backgroundSize: `${theme.modals.jaggedEdgeSize}px`,
+      height: theme.modals.jaggedEdgeSize,
+      position: 'relative',
+      width: '100%',
+      zIndex: mediumZIndex,
+    },
+    dateText: {
+      color: '#A3A3A3',
+      fontSize: normalize(10),
+      fontWeight: '500',
+    },
+    bottomDate: {
+      marginTop: normalize(5),
+    },
+    buttonsRow: {
+      alignItems: 'flex-end',
+      flex: 1,
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+    },
+    rightButton: {
+      marginLeft: 'auto',
+      minWidth: normalize(80),
+    },
+    row: {
+      alignItems: 'center',
+      backgroundColor: 'white',
+      flexDirection: 'row',
+      paddingHorizontal: 0,
+    },
+    title: {
+      justifyContent: 'flex-end',
+      paddingTop: '2em',
+    },
+    leftMargin: {
+      marginLeft: normalize(10),
+    },
+    leftTitle: {
+      color: '#555',
+      flex: 1,
+      fontSize: normalize(20),
+      fontWeight: '700',
+    },
+    rightTitle: {
```

Comment:
it seems that there are several styles definitions duplicated across components and not used (this one in particular is in: `ModalSendEvent`, `ModalReceivEvent`, `ModalWrapper`, `EventStyles`)

Reply:
Deleted _ModalSendEvent_ and _ModalReceivEvent_, as they don't seem to be used anywhere.

Removed unused styles from _ModalWrapper_

Same for _EventStyles_